### PR TITLE
Milestone 5: Testing edit item, update completed, and clear completed

### PR DESCRIPTION
**Pull request by belalgamal46**
### **Testing edit and clear completed**
- Create edited item test file
- Create a test file for update completed value to true
- Create clear completed test file

### CODE REVIEW

File: src/modules/__tests__/AddEditedTaskToStore.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <input type="text" name="description" value="test one" id="0" />
+        <i class="fa-solid fa-trash-can"></i>
+      </li>
+    </ul>
+    `;
+
+    Store.setItems([
+      { index: 0, completed: false, description: 'test one' },
+      { index: 1, completed: false, description: 'test two' },
+    ]);
```
Comment (TadesseDev) [inline src/modules/__tests__/AddEditedTaskToStore.test.js:25]: - [Optional] You are doing great, This is a simple suggestion, as the requirement for the previous project it has been mentioned every task should reflect the length of the array while the task added to the items. This means when inserting task one it should have `index=1`, second task `index=2`, and so on... To be consistent (as you will be testing real data) in your test I suggest you do the same.

**Review by TadesseDev — APPROVED** [review]
### Hi @Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

You did well implementing the project requirement. I have left you a suggestion, other than that you are good to go.
Congratulations! 🎉 

68747470733a2f2f692e67696665722e636f6d2f6f726967696e2f36352f36353164353034326534396532656561363365613934633039653131393731365f773230302e676966

## Highlights
 - [x]  Correct GitHub Flow 👍🏾
 - [x]  Good commit history 🥇
 - [x]  Good PR title/description 💯
 - [x]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/__tests__/AddEditedTaskToStore.test.js (hunk 1 above)
Comment (belalgamal46) [reply src/modules/__tests__/AddEditedTaskToStore.test.js:25]: @Tadesse-Alemayehu In the real code I start the index from zero I made my test code based on that

Comment (TadesseDev) [reply src/modules/__tests__/AddEditedTaskToStore.test.js:25]: Dear @belalgamal46, I can only say this was a primary requirement. You should have gotten a review about that.

**Pull request merged**